test/evergreen.yml

```diff
@@ -48,12 +48,30 @@ functions:
   "configure wiredtiger": &configure_wiredtiger
     command: shell.exec
     params:
-      working_dir: "wiredtiger/build_posix"
+      working_dir: "wiredtiger"
       shell: bash
       script: |
         set -o errexit
         set -o verbose
-        if [ "$OS" != "Windows_NT" ]; then
+        # Check if the build variant has specified a build type, always default to
+        # autoconf if $is_cmake_build is not declared.
```

Comment:
"autoconf --> Autoconf/Libtool" to keep consistent with later comments.

Reply:
Good point, updated!